scrypto-tests/tests/blueprint.rs

```diff
@@ -2,27 +2,22 @@
 
 use scrypto::prelude::*;
```

Comment:
Can we please add some tests about `use` statements here? with and without `scrypto::prelude`.

Reply:
Sure thing, will do!